tasks/main-stable.yml

```diff
@@ -19,6 +19,7 @@
   # according to patch type 'test' or 'stable'.
   grsecurity_urls:
     patch_type: "{{ grsecurity_build_patch_type }}"
+  tags: always
```

Comment:
I believe this will break the `unofficial` scenario—testing locally...

Reply:
Nope, I'm wrong: `molecule test -s unofficial` gets to the build command without issue. Must be the dynamic import logic now, so tags for skipped includes aren't referenced (even via `always`).